.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 7 days with no activity after asking for more info. Comment or this will be closed in 4 days.'
```

Comment:
Triage policy says 14 days not (7 + 4) 11 days.

Reply:
Fixed the timeouts -- the intent is to warn at 10 days and close 4 days later.